feature_layers/display-route-layer/src/main/java/com/esri/samples/display_route_layer/DisplayRouteLayerSample.java

```diff
@@ -0,0 +1,200 @@
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import com.esri.arcgisruntime.symbology.SymbolStyle;
+import javafx.application.Application;
+import javafx.collections.FXCollections;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.application.Platform;
+
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+import javafx.scene.layout.VBox;
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import javafx.scene.text.TextAlignment;
+import javafx.collections.ObservableList;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DisplayRouteLayerSample extends Application {
+
+    private MapView mapView;
+    private Portal portal;
+    private TextField inputTextField;
+    private PortalItem portalItem; // keep loadable in scope to avoid garbage collection
+    private FeatureCollection featureCollection;
+    private VBox vBox;
+    private GridPane legendGridPane;
+    private SymbolStyle symbolStyle;
+
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Display Route Layer Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // authentication with an API key or named user is required to access basemap and other location services
+            String yourAPIKey = System.getProperty("apiKey");
+            ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+            // create a map with the topographical basemap style
+            ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+            // create a map view and set the map to it
+            mapView = new MapView();
+            mapView.setMap(map);
+
+            // set a viewpoint on the map view
+            mapView.setViewpoint(new Viewpoint(45.2281, -122.8309, 57e4));
+
+            // create text field to input user's own portal item ID
+            inputTextField = new TextField("0e3c8e86b4544274b45ecb61c9f41336");
+            inputTextField.setMaxWidth(250);
+
+            //Create a VBox
+            vBox = new VBox();
```

Comment:
```suggestion
            var vBox = new VBox();
```
The VBox can be created as a local variable